docs/platforms/apple/guides/ios/session-replay/performance-overhead.mdx

```diff
@@ -59,3 +60,43 @@ SentrySDK.start(configureOptions: { options in
   options.sessionReplay.sessionSampleRate = if isLowEnd() { 0.0 } else { 0.1 }
 })
 ```
+
+### Enable Experimental View Renderer
+
+<Alert>
+  In case you are noticing issues with the experimental view renderer, please
+  report the issue on [GitHub](https://github.com/getsentry/sentry-cocoa).
+</Alert>
+
+Starting with v8.47.0 you can enable the up to 5x faster new view renderer, reducing the impact of Session Replay on the main thread and potential frame drops.
+
+While we do recommend the new view renderer, it is currently considered an experimental feature to further evaluate its stability.
+After the evaluation phase we are going to enable it by default and eventually replace the default implementation with it in the near future.
+
+```swift
+SentrySDK.start(configureOptions: { options in
+  options.sessionReplay.enableExperimentalViewRenderer = true
+})
+```
+
+### Enable Fast (Incomplete) View Rendering
```

Comment:
I wouldn't document this here. the perf improvement is not that huge and we're confident with the stability of the above. adding this into the mix just makes it confusing and harder for ppl to decide what to do.
it's ok to keep it in code where we also point out it's experimental and produces incomplete renders. if someone really wants to dig into it, they're welcome to.

Reply:
Makes sense, I'll remove the section